torchvision/utils.py

```diff
@@ -61,25 +66,28 @@ def make_grid(
 
     if normalize is True:
         tensor = tensor.clone()  # avoid modifying tensor in-place
-        if range is not None:
-            assert isinstance(range, tuple), \
-                "range has to be a tuple (min, max) if specified. min and max are numbers"
+        if "range" in kwargs.keys():
+            if kwargs["range"] is not None:
+                assert isinstance(kwargs["range"], tuple), \
+                    "range has to be a tuple (min, max) if specified. min and max are numbers"
+            vmin, vmax = get_range(kwargs["range"])
```

Comment:
If `kwargs["range"]` ends up being `None`, I believe this won't work. Might be best if you refactor this a bit to avoid complications.

Reply:
Refer to fmassa's suggestion, the code has been simplified a lot.